.github/workflows/push_docker.yaml

```diff
@@ -20,7 +20,10 @@ on:
       - 'pyproject.toml'
 
 env:
-  REGISTRY_IMAGE: mlenergy/zeus
+
```

Comment:
Please delete the new line. You need to check the changes yourself at least once before requesting review.